test_runner/random_test_runner/src/random_test_runner.cpp

```diff
@@ -113,13 +113,28 @@ TestSuiteParameters RandomTestRunner::collectTestSuiteParameters()
     this->declare_parameter<bool>("ego_goal_partial_randomization", false);
   tp.ego_goal_partial_randomization_distance =
     this->declare_parameter<double>("ego_goal_partial_randomization_distance", 30.0);
-  tp.npcs_count = this->declare_parameter<int>("npc_count", 10);
-  tp.npc_min_speed = this->declare_parameter<double>("npc_min_speed", 0.5);
-  tp.npc_max_speed = this->declare_parameter<double>("npc_max_speed", 3.0);
-  tp.npc_min_spawn_distance_from_ego =
-    this->declare_parameter<double>("npc_min_spawn_distance_from_ego", 10.0);
-  tp.npc_max_spawn_distance_from_ego =
-    this->declare_parameter<double>("npc_max_spawn_distance_from_ego", 100.0);
+  tp.npc_vehicle_count = this->declare_parameter<int>("npc_vehicle_count", 10);
+  tp.npc_vehicle_min_speed = this->declare_parameter<double>("npc_vehicle_min_speed", 0.5);
+  tp.npc_vehicle_max_speed = this->declare_parameter<double>("npc_vehicle_max_speed", 3.0);
+  tp.npc_vehicle_min_spawn_distance_from_ego =
+    this->declare_parameter<double>("npc_vehicle_min_spawn_distance_from_ego", 10.0);
+  tp.npc_vehicle_max_spawn_distance_from_ego =
+    this->declare_parameter<double>("npc_vehicle_max_spawn_distance_from_ego", 100.0);
+  tp.npc_pedestrian_count = this->declare_parameter<int64_t>("npc_pedestrian_count", 1);
+  tp.npc_pedestrian_planner =
+    this->declare_parameter<std::string>("npc_pedestrian_planner", "context_gamma_planner");
+  tp.npc_pedestrian_min_speed = this->declare_parameter<double>("npc_pedestrian_min_speed", 0.1);
+  tp.npc_pedestrian_max_speed = this->declare_parameter<double>("npc_pedestrian_max_speed", 1.0);
+  tp.npc_pedestrian_behavior_static =
+    this->declare_parameter<bool>("npc_pedestrian_behavior_static", true);
+  tp.npc_pedestrian_behavior_crosswalk =
+    this->declare_parameter<bool>("npc_pedestrian_behavior_crosswalk", true);
+  tp.npc_pedestrian_behavior_walk_along_lane =
+    this->declare_parameter<bool>("npc_pedestrian_behavior_walk_along_lane", true);
+  tp.npc_pedestrian_lanelet_min_offset =
+    this->declare_parameter<double>("npc_pedestrian_lanelet_min_offset", 3.0);
+  tp.npc_pedestrian_lanelet_max_offset =
+    this->declare_parameter<double>("npc_pedestrian_lanelet_max_offset", 5.0);
```

Comment:
Please use [generate_parameters library](https://github.com/PickNikRobotics/generate_parameter_library) and simplify your code.